sdk/spring/azure-spring-boot-test-selenium-common/src/main/java/com/azure/test/aad/common/SeleniumITHelper.java

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.test.aad.common;
+
+import com.azure.spring.test.AppRunner;
+import java.io.File;
+import java.io.IOException;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.regex.Pattern;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeDriverService;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class SeleniumITHelper {
+    protected AppRunner app;
+    protected WebDriver driver;
+    protected WebDriverWait wait;
+    protected static Map<String, String> DEFAULT_PROPERTIES = new HashMap<>();
+
+    static {
+        init();
+    }
+
+    private static void init() {
+        final String chromedriverLinux = "chromedriver_linux64";
+        final String chromedriverWin32 = "chromedriver_win32.exe";
+        final String chromedriverMac = "chromedriver_mac64";
+        String classpath = SeleniumITHelper.class.getProtectionDomain().getCodeSource().getLocation().getPath();
+        final String directory = classpath + "driver/";
+        String osName = System.getProperty("os.name").toLowerCase();
+        Process process = null;
+        try {
+            File dir = new File(directory);
+            if (Pattern.matches("linux.*", osName)) {
+                process = Runtime.getRuntime().exec("chmod +x " + chromedriverLinux, null, dir);
+                process.waitFor();
+                System.setProperty(ChromeDriverService.CHROME_DRIVER_EXE_PROPERTY, directory + chromedriverLinux);
+            } else if (Pattern.matches("windows.*", osName)) {
+                System.setProperty(ChromeDriverService.CHROME_DRIVER_EXE_PROPERTY, directory + chromedriverWin32);
+            } else if (Pattern.matches("mac.*", osName)) {
+                process = Runtime.getRuntime().exec("chmod +x " + chromedriverMac, null, dir);
+                process.waitFor();
+                System.setProperty(ChromeDriverService.CHROME_DRIVER_EXE_PROPERTY, directory + chromedriverMac);
+            } else {
+                throw new IllegalStateException("Unrecognized osName. osName = " + System.getProperty("os.name"));
+            }
+        } catch (InterruptedException | IOException e) {
+            throw new RuntimeException(e);
+        } finally {
+            if (process != null) {
+                process.destroyForcibly();
+            }
+        }
+    }
+
+    protected void createDriver() {
```

Comment:
Can we put `createDriver()` and `createAppRunner()` in to the constructor of `SeleniumITHelper`, then it will be called automatically.

Refs: https://stackoverflow.com/questions/2054022/is-it-unnecessary-to-put-super-in-constructor